src/ape_test/providers.py

```diff
@@ -8,6 +8,8 @@
 from ape.types import SnapshotID
 from ape.utils import gas_estimation_error_message
 
+DEFAULT_CHAIN_ID = 61
```

Comment:
I am trying to figure out where this gets set in `eth-tester`.
It isn't released yet but I think they are configuring having this get set more configurably